### PR TITLE
Some improvements for CoRL experiments

### PR DESCRIPTION
**Pull request by alvinsunyixiao**
* The state channel that the linear controller subscribes to is now described with a parameter called `state_channel`.
* rviz config file can be changed in `ff_viz.launch.py` via command arguments.
* Integrates safety filter from #3

### CODE REVIEW

File: ff_control/scripts/safety_filter [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+        # Create timer to check if we've received any messages in the last 'wait_period' seconds
+        self.timer = self.create_timer(0.1, self.check_timer_callback)
+
+        # Create a subscriber to listen for a kill message
+        self.kill_sub = self.create_subscription(
+            Bool,
+            'kill',
+            self.kill_callback,
+            10)
```
Comment (somritabanerjee) [inline ff_control/scripts/safety_filter:70]: Is there a publisher for the 'kill' channel? I didn't see one in the codebase and don't know of a universal trigger for publishing to 'kill'. Or is the idea to manually publish from the terminal?

File: ff_control/src/linear_ctrl.cpp [hunk 2]
```diff
@@ -59,8 +59,11 @@ LinearController::LinearController()
 : rclcpp::Node("linear_ctrl_node"),
   WrenchController()
 {
+  this->declare_parameter("state_channel", "est/state");
   state_sub_ = this->create_subscription<FreeFlyerStateStamped>(
-    "est/state", 10, std::bind(&LinearController::StateCallback, this, _1));
+    this->get_parameter("state_channel").as_string(),
```
Comment (somritabanerjee) [inline ff_control/src/linear_ctrl.cpp:64]: .get_parameter_value() needed here?

**Review by somritabanerjee — COMMENTED** [review]
Looks good! I just had a couple comments.

File: ff_control/src/linear_ctrl.cpp (hunk 2 above)
Comment (alvinsunyixiao) [reply ff_control/src/linear_ctrl.cpp:64]: I don't think so. It has a different API from Python.

**Pull request merged**